packages/lit-html/src/lit-html.ts

```diff
@@ -982,13 +982,14 @@ class Template {
     // We could set walker.currentNode to another node here to prevent a memory
     // leak, but every time we prepare a template, we immediately render it
     // and re-use the walker in new TemplateInstance._clone().
-    debugLogEvent?.({
-      kind: 'template prep',
-      template: this,
-      clonableTemplate: this.el,
-      parts: this.parts,
-      strings,
-    });
+    debugLogEvent &&
```

Comment:
Can we file a Terser bug on this? If you can eliminate `&&`, in null cases you can eliminate `?.`

Reply:
Filed as https://github.com/terser/terser/issues/1400